processing.js

```diff
@@ -2497,7 +2497,7 @@ module.exports = function(options, undef) {
      * @param {float} sy  the amount to scale on the y-axis
      */
     scale: function(sx, sy) {
-      if (sx && !sy) {
+      if (sx && sy === undef) {
```

Comment:
This was added because `sy` and `sz` can be 0, which is not the same as not having been passed into the function.